filebeat/_meta/common.full.p2.yml

```diff
@@ -82,6 +82,11 @@ filebeat.prospectors:
   # This is especially useful for multiline log messages which can get large.
   #max_bytes: 10485760
 
+  # Maximum depth for "**" patterns. This setting is disabled by default, i.e.,
```

Comment:
I renamed to `recursive_glob_depth` for now, I don't think we should encourage users to use more than one per path so the namespace seems overkill to me.